recipes-kernel/linux/overlays/verdin-imx8mm_gt911_overlay.dts

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2
+/*
+ * Copyright 2020-2021 Opentrons
+ */
+
+// Verdin DSI to LVDS Adapter with connected ATM0700L61-CT display (7 inch) with a
+// resolution of 1024x600 pixel. Semicustom from AZ Displays.
+
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "toradex,verdin-imx8mm";
+};
+
+
+&i2c4 {
+      clock-frequency = <100000>;
+      #address-cells = <1>;
+      #size-cells = <0>;
+      gt911: touch@5d {
+        compatible = "goodix,gt911";
+        reg = <0x5d>;
+        pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_gpio_9_dsi>, <&pinctrl_i2s_2_bclk_touch_reset>;
```

Comment:
indentation for this block might be a little off at some places, think the device tree syntax doesn't really care. Just might throw off people trying to go off of this to write some other overlay.